src/core/client/stream/tabs/Comments/Stream/FeaturedComments/FeaturedBy.css

```diff
@@ -0,0 +1,14 @@
+$status-modal-username: $colors-mono-900;
```

Comment:
I don't see this used in this file. Should it be removed?

Reply:
✅